### PR TITLE
PR for tutorial on new feature

### LINKED ISSUE #358: Create Tutorial - Learn to add a new feature.
## Skills covered

Codebase Navigation,
Figure out what code to change,
Unit Tests,
How to make a full stack change (Backend & Frontend)

## Similar issues

[19971](https://github.com/oppia/oppia/issues/19971), [19864](https://github.com/oppia/oppia/issues/19864), [19852](https://github.com/oppia/oppia/issues/19852), [19850](https://github.com/oppia/oppia/issues/19850), [19681](https://github.com/oppia/oppia/issues/19681)

## Existing relevant pages

[oppia/wiki/Find-the-right-code-to-change](https://github.com/oppia/oppia/wiki/Find-the-right-code-to-change)

[oppia/wiki/Tests](https://github.com/oppia/oppia/wiki/Tests)

[oppia/wiki/Launching-new-features](https://github.com/oppia/oppia/wiki/Launching-new-features)

## Example

Implement a new feature that recommends explorations to learners based on their past activity or interests.

This will involve: Creating a UI element to display recommendations (Frontend)
Recommendation algorithms (simple ones), fetching relevant data from learner profiles and exploration metadata.

### PR DESCRIPTION
**Pull request by Ash-2k3**
Fix https://github.com/oppia/oppia-web-developer-docs/issues/358

Link to Google Doc: https://docs.google.com/document/d/19JEsUg0JErf6wHOtCbkmfPuasC7veSuG56h7cOQAbC0/edit?tab=t.0#heading=h.f612ojb7j8if

### CODE REVIEW

File: Tutorial-Learn-How-to-Add-a-New-Feature.md [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+The goal is to create a Welcome Modal for the Contributor Dashboard that appears when a user visits the page for the first time. The modal will display the following static text:
+
+*“Welcome to the Contributor Dashboard. We appreciate your help in making the Oppia lessons better. Thanks!”*
+
+After the user closes the modal, they will be able to proceed with their usual tasks. The modal will not appear again on future visits.
+
+## Prerequisites
+
+Before you begin, ensure you have:
+- Set up your development environment. (If you haven't, follow the [Oppia setup instructions](https://github.com/oppia/oppia/wiki/End-to-End-Tests)).
```
Comment (U8NWXD) [inline Tutorial-Learn-How-to-Add-a-New-Feature.md]: I think this is the wrong link?

**Review by U8NWXD — CHANGES_REQUESTED** [review]
Please also add this tutorial to the sidebar

File: Tutorial-Learn-How-to-Add-a-New-Feature.md (hunk 1 above)
Comment (Ash-2k3) [reply Tutorial-Learn-How-to-Add-a-New-Feature.md]: Fixed, thanks for catching

**Comment by Ash-2k3** [comment]
Hi @U8NWXD, I have addressed your comments, PTAL!

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**